multistate.go

```diff
@@ -216,7 +216,7 @@ func (m *Multistate) GetStateActions(ctx context.Context, state uint64) []string
 	return nil
 }
 
-func (m *Multistate) DoAction(ctx context.Context, entity Entity, action string, opts ...interface{}) (uint64, error) {
+func (m *Multistate) DoAction(ctx context.Context, entity Entity, id interface{}, action string, opts ...interface{}) (uint64, error) {
```

Comment:
`id` is already in entity, you don't need a separate field.

Reply:
But we can't  get it